### PR TITLE
[jasmine] Clock method 'install' returns Clock.

### PR DESCRIPTION
**Pull request by unicornfingers**
The Clock `install` method [returns itself](https://github.com/jasmine/jasmine/blob/v3.6.0/src/core/Clock.js#L51), so its return type should be `Clock` rather than `void`.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jasmine/jasmine/blob/v3.6.0/src/core/Clock.js#L51

### CODE REVIEW

**Review by devoto13 — APPROVED** [review]
LGTM

**Review by chivesrs — APPROVED** [review]

**Comment by unicornfingers** [comment]
Hi there! Thanks for the quick reviews. I'm not sure if I need _all three_ approvals before I can merge this PR, or if I _should_ be able to merge with the two approvals it already has?

If the latter, well, Github won't let me merge yet even though I have 2 approvals? 🤷 

If the former, I'll just be patient! 😄

**Comment by devoto13** [comment]
See a message from the bot above:

> Because this is a widely-used package, a DT maintainer will need to review it before it can be merged.

So we need to wait until DT maintainer merges it.

**Review by djungowski — APPROVED** [review]

**Review by kolodny — APPROVED** [review]

**Review by FDIM — APPROVED** [review]

**Pull request merged**
